packages/@aws-cdk/region-info/README.md

```diff
@@ -0,0 +1,46 @@
+## AWS region information, such as service principal names
+### Usage
+Information pertaining to an AWS Region can be obtained using the `RegionInfo`
+class:
+
+```ts
+const dublin = RegionInfo.forRegion('eu-west-1');
+```
+
+### User-supplied region information
+The library comes pre-loaded with information about all publicly available AWS
+regions as of the release date of a particular version. It is possible this
+information is incomplete (or possibly incorrect). A user can provide their own
+region information by placing data in a JSON document at
+`~/.cdk/region-info.json`. The location of the file can be customized either by
+setting the `CDK_REGION_INFO_PATH` environment variable, or by setting the
+`RegionInfo.userDataPath` property.
+
+The file must contain a single region information object, or an array of region
+information objects, which must conform to the JSON-Schema found in this
+package's `schema/region-info.schema.json` file, for example:
+
+```json
+[
+  {
+    // Obviously not a "real" region:
```

Comment:
use "```js" so that markdown formatting will be okay with the comments

Reply:
Ah - it showed properly on my IDE... Whoops.